res/enkf/es_update.py

```diff
@@ -60,9 +60,10 @@ def analysis_smoother_update(
 
     _lib.update.copy_parameters(source_fs, target_fs, ensemble_config, ens_mask)
 
+    reports = []
```

Comment:
There should be a link to which ministep it came from. The ministeps have names, so it should at least be a dict